src/sidebar/store/create-store.js

```diff
@@ -22,9 +101,10 @@ import { createReducer, bindSelectors } from './util';
  * each action and selector from the input modules as a method which operates on
  * the store.
  *
- * @param {Object[]} modules
+ * @param {Module<any,any,any,any>[]} modules
  * @param {any[]} [initArgs] - Arguments to pass to each state module's `init` function
  * @param {any[]} [middleware] - List of additional Redux middlewares to use
+ * @return Store<any,any,any>
  */
```

Comment:
Ideally the returned `Store` would have a type that depends upon `modules`. I haven't worked out how to do that yet. The tricky part is that the number of modules is variable and the type of each module is different.